README.md

```diff
@@ -177,6 +180,14 @@ const alt = new Alt();
 
 ### ES6
 
+##Typescript Definitions
+The typescript definitions for alt are located in the typings directory. This should be included in your project under typings/alt or whatever folder you use to manage your definitions files. You can import the dependencies react, flux, and eventemitter3 easily with [TSD](https://github.com/DefinitelyTyped/tsd). From here you can reference your typings as per usual with a reference tag ```<reference path="<path>.d.ts" />```. Check the [alt-typescript-tutorial](https://github.com/Shearerbeard/alt-typescript-tutorial) for more information and project examples.
```

Comment:
If event emitter and flux dependencies are no longer required it would solve our problem with installing the typings directly npm and bower - both were used as an interface since alt seemed to provide some options for accessing and overriding the default implementations.